src/NzbDrone.Core/Notifications/PushBullet/PushBulletProxy.cs

```diff
@@ -151,6 +151,11 @@ private void SendNotification(string title, string message, RestRequest request,
                 request.AddParameter("title", title);
                 request.AddParameter("body", message);
 
+                if (!String.IsNullOrEmpty(settings.SenderId))
```

Comment:
That method doesn't exist

![image](https://cloud.githubusercontent.com/assets/1753736/12657488/6f5132a0-c5fb-11e5-9a73-b7afe6086751.png)